app/services/redmine_request_json.rb

```diff
@@ -0,0 +1,28 @@
+class RedmineRequestJson
+  def initialize(offset)
+    @offset = offset
+    @res = nil
+    @response_body = nil
+  end
+
+  def fetch_json_from_redmine
+    http_request
+    extract_request_time_entries
+    @response_body
+  end
+
+  private
+
+  def http_request
+    uri = URI('https://redmine.renuo.ch/time_entries.json?')
+    params = { limit: '100', offset: (@offset * 100).to_s, key: ENV['REDMINE_API_KEY'] }
```

Comment:
What happens if `REDMINE_API_KEY` env is not set? And: is there a test for what happens then?